test/specs/run-spec.js

```diff
@@ -35,8 +35,8 @@ function runSpecs(title, dir, showCompletionTable, options) {
   });
 }
 
-runSpecs('GFM', './gfm', true, { gfm: true });
-runSpecs('CommonMark', './commonmark', true, { headerIds: false });
-runSpecs('Original', './original', false, { gfm: false });
+runSpecs('GFM', './gfm', true, { gfm: true, pedantic: false, headerIds: false });
+runSpecs('CommonMark', './commonmark', true, { gfm: false, pedantic: false, headerIds: false });
```

Comment:
I thought `pedantic` was default false so why assign it here?

Reply:
Ya I thought it was better to be verbose so it is easier to know what is going on, and in case the defaults change in the future.